### ISSUE TITLE
Playstate of Spotty not synced to Android App

### ISSUE THREAD

**Issue by BerniPi** [michaelherger/Spotty-Plugin#113]
Hello,

I can see the Spotty - players in my Spotify App and play some music from there. Then i can control the music from the app and from lms and the informations are in snyc.
But if it start a playlist, artist, song from lms webinterface, i dont get any infirmation int Spotify Android app. It's even possible to play independently music on the phone with the same account.

Is this expected behaviour?

**Comment by michaelherger** [comment]
Yes, that's what I call LMS or Spotty mode, vs. Connect mode where you have full control from the Spotify universe.

**Comment by michaelherger** [comment]
Once Connect mode enabled you have to control your SB through the Spotify app on your mobile device or computer, _not_ through LMS.

**Comment by AlphaCraft9658** [comment]
> Once Connect mode enabled you have to control your SB through the Spotify app on your mobile device or computer, _not_ through LMS.

There lies the issue. When controlling via the Spotify app, once I pause the playback, it jumps completely to the end of the song.

**Comment by michaelherger** [comment]
See https://github.com/michaelherger/Spotty-Plugin/issues/102, #82, and a few more. Connect mode is known to be fragile.

**Comment by AlphaCraft9658** [comment]
> See https://github.com/michaelherger/Spotty-Plugin/issues/102, #82, and a few more. Connect mode is known to be fragile.

I see. It's fine for me to just run in LMS mode. It works fine and I even found an app for controlling the radio, which makes it even easier. Thank you for your efforts! Great plugin!

**Comment by michaelherger** [comment]
With the recent changes Spotify applied to their infrastructure I'll have to re-implement Connect mode from scratch. I therefore decided to remove that feature from the current release. Let's collect new reports once I have found a way to re-implement Connect.

Thanks for your understanding.
